tests/helloworld.at

```diff
@@ -0,0 +1,15 @@
+AT_BANNER([p4c compiler driver unit tests])
+
+AT_SETUP([p4c])
+
+m4_define([P4C_CHECK], [
+AT_CHECK([p4c -t tofino min.p4], [1],
```

Comment:
I am working on a compiler driver called p4c which is analogous to clang or gcc. I will push the code for p4c before this PR goes in.